discoverable-client/src/main/resources/static/index.html

```diff
@@ -82,8 +82,8 @@ <h2>WebSockets</h2>
 <!-- Bootstrap core JavaScript
 ================================================== -->
 <!-- Placed at the end of the document so the pages load faster -->
-<script type="text/javascript" src="webjars/jquery/3.6.0/jquery.min.js"></script>
-<script type="text/javascript" src="webjars/bootstrap/4.3.1/js/bootstrap.min.js"></script>
+<script type="text/javascript" src="https://cdn.jsdelivr.net/npm/jquery@3.7.1/dist/jquery.min.js"></script>
+<script type="text/javascript" src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/js/bootstrap.bundle.min.js"></script>
```

Comment:
what happens if requests to the CDN fail to download? Does it affect integration tests for example?

Reply:
No, it doesn't. This page is only for manual testing.